moodle/pkg/utils/utils.go

```diff
@@ -133,12 +134,17 @@ func (u *Utils) EnsurePluginsInstalled(moodle *operatorv1.Moodle,
 	defer mutex.Unlock()
 	fmt.Println("Inside EnsurePluginsInstalled")
 
-	plugins := moodle.Spec.Plugins
 	erredPlugins := make([]string, 0)
-	for _, pluginName := range plugins {
+
+	for _, pluginName := range supportedPlugins {
 		fmt.Printf("  Installing plugin %s\n", pluginName)
-		pluginInstallDetails := pluginData[pluginName]
+		pluginInstallDetails, ok := pluginData[pluginName]
 
+		if !ok {
```

Comment:
Yeah, as we discussed this is probably dead code as we are already looping over supportedPlugins.

We can remove this.